openshift/process_template.go

```diff
@@ -134,7 +135,12 @@ func LoadProcessedTemplates(ctx context.Context, config Config, username string,
 		token := goajwt.ContextJWT(ctx)
 		if token != nil {
 			vars["OSIO_TOKEN"] = token.Raw
+			id := token.Claims.(jwt.MapClaims)["sub"]
```

Comment:
https://github.com/fabric8-services/fabric8-wit/blob/master/log/context.go#L14 ?

Reply:
well, I didn't use the `extractIdentityID` function since we already have the token, and we don't need an error (which is why a tuple is returned by `extractIdentityID`)

Reply:
ok, makes sense

Reply:
so what happens when you don't have the token ? I don't see why you wouldn't re-use this utility function.

Reply:
I just don't set it.
I can reuse the utility, but it will do the JWT token extraction twice then.

Reply:
ok